.grabl/automation.yml

```diff
@@ -61,10 +61,64 @@ build:
+        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
         bazel test //tests/behaviour/concept/... --test_output=errors --jobs=1
-        bazel test //tests/behaviour/graql/language/get/... --test_output=errors --jobs=1
-      # TODO: add other Graql Language tests once they stabilise fully
-      # TODO: remove --jobs=1 from Concept and Graql tests once Grakn runner is parallelisable
+    test-behaviour-match:
+      image: graknlabs-ubuntu-20.04
+      type: foreground
+      command: |
+        pyenv global 3.6.10
+        pip install -r requirements_dev.txt
+        sudo unlink /usr/bin/python3
+        sudo ln -s $(which python3) /usr/bin/python3
+        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
+        export ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME
+        export ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD
+        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
+        bazel test //test/behaviour/graql/language/match/... --test_output=streamed
+        bazel test //test/behaviour/graql/language/get/... --test_output=streamed
+    test-behaviour-writable:
+      image: graknlabs-ubuntu-20.04
+      type: foreground
+      command: |
+        pyenv global 3.6.10
+        pip install -r requirements_dev.txt
+        sudo unlink /usr/bin/python3
+        sudo ln -s $(which python3) /usr/bin/python3
+        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
+        export ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME
+        export ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD
+        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
+        bazel test //test/behaviour/graql/language/insert/... --test_output=streamed
+        bazel test //test/behaviour/graql/language/delete/... --test_output=streamed
+        bazel test //test/behaviour/graql/language/update/... --test_output=streamed
+    test-behaviour-definable:
+      image: graknlabs-ubuntu-20.04
+      type: foreground
+      command: |
+        pyenv global 3.6.10
+        pip install -r requirements_dev.txt
+        sudo unlink /usr/bin/python3
+        sudo ln -s $(which python3) /usr/bin/python3
+        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
+        export ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME
+        export ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD
+        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
+        bazel test //test/behaviour/graql/language/define:checkstyle --test_output=streamed
+        bazel test //test/behaviour/graql/language/define:test-core --test_output=streamed --jobs=1
+        bazel test //test/behaviour/graql/language/undefine:checkstyle --test_output=streamed
+        bazel test //test/behaviour/graql/language/undefine:test-core --test_output=streamed --jobs=1
```

Comment:
Why are these split up? We can simply write `bazel test //test/behaviour/graql/language/define/...` and the same for undefine. With `--jobs=1` of course. Also there is no need for streamed test output - `--test_output=errors` should be enough.

(For the record, streamed test output results in job logs easily exceeding 10,000 lines)

Reply:
Split up due to cluster tests not presently functioning. Otherwise addressed